mxd/backend-service/src/main/java/org/eclipse/mxd/service/Impl/ContentServiceImpl.java

```diff
@@ -0,0 +1,165 @@
+public class ContentServiceImpl implements ContentService {
+
+    private static final Logger logger = Logger.getLogger(ContentServiceImpl.class.getName());
+
+    private final ContentsRepository contentServiceRepository = new ContentsRepositoryImpl();
+
+    @Override
+    public Response getAll(UriInfo uriInfo) {
+        try {
+            List<ContentsRequest> contentResponse = this.contentServiceRepository.getAllAssets();
+            if (contentResponse != null && !contentResponse.isEmpty()) {
+                List<ContentModelResponse> contentsModels = new ArrayList<ContentModelResponse>();
+                for (ContentsRequest contentsRequest : contentResponse) {
+                    JsonNode jsonNode = new ObjectMapper().readTree(contentsRequest.getAsset());
+                    ContentModelResponse contentsModelRes = new ContentModelResponse();
+                    contentsModelRes.setId(contentsRequest.getId());
+                    contentsModelRes.setAsset(jsonNode);
+                    contentsModelRes.setUrl(uriInfo.getAbsolutePath().toString() + contentsRequest.getId());
+                    contentsModelRes.setCreatedDate(contentsRequest.getCreatedDate());
+                    contentsModelRes.setUpdatedDate(contentsRequest.getUpdatedDate());
+                    contentsModels.add(contentsModelRes);
+                }
+                return Response.ok(new ObjectMapper().writeValueAsString(contentsModels)).build();
+
+            } else {
+                ErrorModel errorModel = new ErrorModel(404, "Resource Not Found", "Requested Resource Not Found");
+                return Response.status(Response.Status.NOT_FOUND)
+                        .entity(new ObjectMapper().writeValueAsString(errorModel))
+                        .type(MediaType.APPLICATION_JSON)
+                        .build();
+            }
+        } catch (Exception e) {
+            try {
+                logger.info(e.getMessage());
+                ErrorModel errorModel = new ErrorModel(500, "Internal Server Error", e.getMessage());
+                return Response.status(Response.Status.INTERNAL_SERVER_ERROR)
+                        .entity(new ObjectMapper().writeValueAsString(errorModel))
+                        .type(MediaType.APPLICATION_JSON)
+                        .build();
+            } catch (Exception ex) {
+                logger.info(ex.getMessage());
+                return Response.status(Response.Status.INTERNAL_SERVER_ERROR)
+                        .entity("Internal Server Error")
+                        .type(MediaType.APPLICATION_JSON)
+                        .build();
+            }
+        }
+
+
+    }
+
+    @Override
+    public Response getById(Long id, UriInfo uriInfo) {
+        try {
+            ContentsRequest content = this.contentServiceRepository.getAssetById(id);
+            if (content != null && content.getId() != null) {
+                ContentJsonResponse contentModelRes = new ContentJsonResponse();
```

Comment:
`ContentJsonResponse` is not needed, directly parse into a JsonNode and use it.

Reply:
passed the Json Node